itou/www/siae_evaluations_views/views.py

```diff
@@ -725,11 +725,9 @@ def siae_submit_proofs(request, evaluated_siae_pk):
 
     messages.success(
         request,
-        mark_safe(
-            "<b>Justificatifs transmis !</b><br>"
-            "Merci d'avoir pris le temps de transmettre vos pièces justificatives.<br>"
-            "Le contrôle de celles-ci est à la charge de votre DDETS maintenant, vous serez notifié du résultat "
-            "(qu'il soit positif ou négatif) par mail lorsque celui-ci sera finalisé."
+        (
+            "Justificatifs transmis !|| Leur contrôle est à la charge de votre DDETS."
```

Comment:
L'espace est en trop, non ?
```suggestion
            "Justificatifs transmis !||Leur contrôle est à la charge de votre DDETS."
```